scripts/generate-changelog-data.mjs

```
@@ -0,0 +1,41 @@
+const sourceFile = path.join(repoRoot, CHANGELOG_SOURCE_PATH);
+const outputFile = path.join(repoRoot, CHANGELOG_DATA_PATH);
+
+const data = buildChangelogData(readFileSync(sourceFile, "utf-8"));
+writeFileSync(outputFile, serializeChangelogData(data), "utf-8");
+
+const bullets = data.releases.reduce(
+  (total, release) => total + release.categories.reduce((sum, category) => sum + category.items.length, 0),
+  0,
+);
+
+console.log(`Generated ${outputFile}`);
```

ESLint

[error] 'console' is not defined.

(no-undef)